src/components/PageHeadingFlowRuns.vue

```diff
@@ -1,14 +1,24 @@
 <template>
   <page-heading class="page-heading-flow-runs" :crumbs="crumbs">
-    <template #actions>
+    <template v-if="!hideActions" #actions>
       <SavedFilters />
     </template>
   </page-heading>
 </template>
 
 <script lang="ts" setup>
+  import { defineProps, withDefaults, computed } from 'vue'
   import PageHeading from '@/components/PageHeading.vue'
   import  SavedFilters from '@/components/SavedFilters.vue'
 
+
+  const props = withDefaults(defineProps<{
+    hideActions?: boolean,
+  }>(), {
+    hideActions: false,
+  })
```

Comment:
no need to define a default for this, as the default is (and always should be) falsey
```suggestion
  const props = defineProps<{
    hideActions?: boolean,
  }>()
```